packages/generators/src/index.ts

```diff
@@ -18,6 +18,9 @@ class GeneratorsCommand {
             },
             [],
             async (outputPath) => {
```

Comment:
can just do default param?

```suggestion
            async (outputPath = process.cwd()) => {
```

Reply:
Yup we can but I personally don't like calling functions as default parameter 😄, looks hacky to me.